### PR TITLE
Add Recipe for minimal vtk build

### PR DESCRIPTION
**Pull request by j-fu**
Hi, this is the pull request for VTK without HDF5 and MPI.
Building will take some time as 1700+ files  are compiled...

### CODE REVIEW

File: V/VTKMinimal/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+cd build
+cmake -DVTK_CUSTOM_LIBRARY_SUFFIX="" -DVTK_USE_SYSTEM_LIBRARIES:BOOL=ON -DVTK_USE_SYSTEM_EIGEN3:BOOL=OFF  -DVTK_USE_SYSTEM_THEORA:BOOL=OFF   -DVTK_USE_SYSTEM_EIGEN=OFF -DVTK_USE_SYSTEM_DOUBLECONVERSION=OFF  -DVTK_USE_SYSTEM_LZMA:BOOL=ON -DVTK_USE_SYSTEM_OGG:BOOL=OFF   -DVTK_USE_SYSTEM_PNG:BOOL=ON  -DVTK_USE_SYSTEM_LZ4=ON -DVTK_USE_SYSTEM_JPEG:BOOL=ON -DVTK_USE_SYSTEM_TIFF:BOOL=ON -DVTK_USE_SYSTEM_ZLIB:BOOL=ON -DVTK_Group_Rendering:BOOL=OFF -DVTK_Group_StandAlone:BOOL=OFF -DModule_vtkCommonCore:BOOL=ON -DModule_vtkCommonDataModel:BOOL=ON -DModule_vtkCommonExecutionModel:BOOL=ON -DModule_vtkCommonMisc:BOOL=ON -DModule_vtkCommonSystem:BOOL=ON -DModule_vtkCommonMath:BOOL=ON -DModule_vtkCommonTransforms:BOOL=ON -DModule_vtkRenderingCore:BOOL=ON -DModule_vtkRenderingContext2D:BOOL=ON -DModule_vtkFiltersCore:BOOL=ON -DModule_vtkFiltersGeneral:BOOL=ON -DModule_vtkFiltersGeometry:BOOL=ON -DModule_vtkFiltersExtraction:BOOL=ON -DModule_vtkFiltersFlowPaths:BOOL=ON -DModule_vtkInteractionWidgets:BOOL=ON -DModule_vtkIOMovie:BOOL=ON -DModule_vtkParallelCore:BOOL=ON -DModule_vtkInteractionStyle:BOOL=ON -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_BUILD_TYPE=Release  ../VTK-8.2.0/
```
Comment (giordano) [inline V/VTKMinimal/build_tarballs.jl]: You should pass also the toolchain file, see the [build tips](https://github.com/JuliaPackaging/Yggdrasil/wiki/Tips-for-Building-Packages#cmake-builds):
```
cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
```

Comment (j-fu) [reply V/VTKMinimal/build_tarballs.jl]: .. vtk complains about missing vtk compile tools then (I think that's why I removed that flag
anyway, and then forgot).

Supporting this appearantly was introduced shortly after the 8.2 release.
https://gitlab.kitware.com/vtk/vtk/-/commit/d67d1059ac2e2da69e5e0b5543a7c2d6808160d0

So we would get this only from git. 

They say that 9.0.0-rc1 will be out soon. So we might start to deploy for 64bit only and make
an update later ?

Comment (j-fu) [reply V/VTKMinimal/build_tarballs.jl]: Edit: hopeful to build now with 9.0.0-rc1

Comment (j-fu) [reply V/VTKMinimal/build_tarballs.jl]: ... Hopes crashed: VTK builds, but need to learn how to export the xcompilation tools built by vtk to client codes

**Comment by j-fu** [comment]
Closing this for now - will come back later after figured out cross-compilation issue

**Pull request closed without merging**